.circleci/config.yml

```diff
@@ -55,6 +52,9 @@ jobs:
       steps:
          - checkout
          - run: git config --global --unset url.ssh://git@github.com.insteadof
+         - run:
```

Comment:
the `git config` commands here can be removed as they're already part of develop

Reply:
I didn't mean to change the circleci config at all and pushed a version restored from develop. Perhaps it was outdated, reverting.